examples/pgconfig/templates/configmap.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    creationTimestamp: null
+    name: geoserver-datasources-env
+data:
+    values.yml: |-
+        backend.pgconfig: true
+        pgconfig.jndi-name: java:comp/env/jdbc/pgconfig
+        pgconfig.schema: public
+        gwc.enabled: false
+        logging.level.org.springframework.retry.support: info
```

Comment:
* `backend.pgconfig: true` is already set by the `pgconfig` spring profile in 
```
SPRING_PROFILES_ACTIVE:
    value: standalone,pgconfig
```
* I would change the default schema from `public` to `pgconfig` or so. Flyway will create it if it doesn't exist. (also change it in `jndi.datasources.pgconfig.schema` below.

* No need to disable gwc nor set that logging level

Reply:
done